apps/dashboard/src/components/project-tools/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@crowdsignal/components';
+import { STORE_NAME } from 'data';
+import { useDispatch, withSelect } from '@wordpress/data';
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ProjectTools = ( { projectId, isFetching } ) => {
+	const { saveAndUpdateProject } = useDispatch( STORE_NAME );
+
+	const syncProject = () => {
+		saveAndUpdateProject( projectId, {
+			title: 'this is here to create a the mock project',
+		} );
+	};
+
+	const publishProject = () => {
+		saveAndUpdateProject( projectId, {
+			title: 'Published!',
+			publish: true,
+		} );
+	};
+
+	return (
+		<div className="project-tools">
+			<Button onClick={ syncProject } disabled={ isFetching }>
+				{ __( 'Save draft', 'dashboard' ) }
+			</Button>
+			<Button primary onClick={ publishProject } disabled={ isFetching }>
+				{ __( 'Publish', 'dashboard' ) }
+			</Button>
+		</div>
+	);
+};
+
+export default withSelect( ( select ) => {
```

Comment:
Nit: I think `useSelect` is the preferred/cleaner way of doing it nowadays. It's what `withSelect` does under the hood anyway.

That said,... this whole component is kind of temporary until we implement autosave so feel free to disregard the comment and keep in mind for future stuff ;)

Reply:
TBH, I initially tried with useSelect but it didn't do the job. That said, I was doing so many changes that maybe my initial use of `useSelect` was flawed. Will try again.